docusaurus/docs/Android/03-ui/02-general-customization/02-chatui.mdx

```diff
@@ -37,14 +40,31 @@ As shown below, by default the SDK provides 5 built-in reactions.
 
 You can override the default set of reactions. In order to define a custom set of reactions for your app, you need to pass the the following data to the `ChatUI.supportedReactions` property:
 
-```kotlin 
+<Tabs>
+<TabItem value="kotlin" label="Kotlin">
+
+```kotlin
 val loveDrawable = ContextCompat.getDrawable(this, R.drawable.ic_reaction_love)!!
 val loveDrawableSelected = ContextCompat.getDrawable(this, R.drawable.ic_reaction_love)!!.apply { setTint(Color.RED) }
```

Comment:
This example looks incorrect without mutating the drawable.

Reply:
I'll create a separate task for that